web/components/data/FcDetailStudies.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="fc-detail-studies">
+    <v-progress-linear
+      v-if="loading"
+      indeterminate />
+    <div
+      v-else-if="studySummaryUnfiltered.length === 0"
+      class="my-8 py-12 secondary--text text-center">
+      There are no studies for this location,<br>
+      please request a study if necessary
+    </div>
+    <template v-else>
+      <div
+        v-for="item in items"
+        :key="item.category.id"
+        class="ml-5">
+        <div class="align-center d-flex mb-4 pr-5">
+          <div class="body-1 flex-grow-1 flex-shrink-1">
+            <div v-if="item.category.studyType === null">
+              Unknown
+            </div>
+            <div v-else>
+              {{item.category.studyType.label}}
+            </div>
+            <div class="mt-2 secondary--text">
+              <span v-if="item.mostRecent !== null">
+                Most Recent
+                {{item.mostRecent.startDate | date}} ({{item.mostRecent.startDate | dayOfWeek}})
+                &#x2022;
+                <span v-if="item.mostRecent.duration !== null">
+                  {{item.mostRecent.duration | durationHuman}} ({{item.mostRecent.duration}} hrs)
+                </span>
+                <span
+                  v-else-if="item.mostRecent.hours !== null"
+                  :title="item.mostRecent.hours.hint">
+                  {{item.mostRecent.hours.description}}
+                </span>
+              </span>
+            </div>
+          </div>
+          <div class="fc-studies-n flex-grow-0 flex-shrink-0 mr-8">
+            <div class="display-2">
```

Comment:
same for this - e.g. `<FcTextSummaryFraction>`?  that would allow us to simplify the detail and aggregate components considerably.